Resources/Prototypes/_SpaceCats/Research/experimental.yml

```
@@ -20,4 +20,28 @@
+
+- type: technology
+  id: ThermalVisionTech
+  name: research-technology-thermal-vision
+  icon:
+    sprite: _SpaceCats/Clothing/Eyes/Goggles/thermal.rsi
+    state: icon
+  discipline: Experimental
+  tier: 2
+  cost: 20000
+  recipeUnlocks:
+  - ClothingEyesThermalVisionGoggles
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)